src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -44,9 +44,11 @@ export class AddBulkConcepts extends Component {
     this.props.addExistingBulkConcepts({ url, data: { data: { expressions: cielConcepts } } });
     window.history.back();
   };
+
   handleCielClick = () => {
     this.props.fetchCielConcepts();
   };
+
```

Comment:
Is this part of this ticket?

Reply:
It's formatting from the linter. it now removed